scripts/build.sh

```diff
@@ -4,6 +4,12 @@ set -e
 
 CLEANSLATE="node_modules/cleanslate/docs/files/cleanslate.css"
 TRIDACTYL_LOGO="src/static/logo/Tridactyl_64px.png"
+BROWSER="firefox"
+
+if [ $1 = "chrome" ]; then
```

Comment:
If called with no arguments, we get `scripts/build.sh: line 9: [: =: unary operator expected` here